src/main/java/com/twilio/oai/resolver/common/CodegenParameterResolver.java

```diff
@@ -10,14 +10,22 @@
 public class CodegenParameterResolver extends Resolver<CodegenParameter> {
 
     private final IConventionMapper mapper;
-    private List<? extends LanguageDataType> languageDataTypes;
-    private final CodegenParameterDataTypeResolver codegenParameterDataTypeResolver = new CodegenParameterDataTypeResolver();
+    private final CodegenParameterDataTypeResolver codegenParameterDataTypeResolver;
     private final CodegenParameterContainerDataTypeResolver codegenParameterContainerDataTypeResolver;
 
     public CodegenParameterResolver(IConventionMapper mapper,
                                     List<? extends LanguageDataType> languageDataTypes) {
         this.mapper = mapper;
-        this.languageDataTypes = languageDataTypes;
+        codegenParameterDataTypeResolver = new CodegenParameterDataTypeResolver(mapper);
```

Comment:
Same comment about this constructor calling the new one.

Reply:
Calling a constructor via constructor to avoid code duplication.
But I think it is better to pass CodegenParameterDataTypeResolver from calling class.